BlueDBOnDisk/src/main/java/org/bluedb/disk/collection/task/UpdateMultipleTask.java

```diff
@@ -28,33 +31,41 @@ public UpdateMultipleTask(BlueCollectionOnDisk<T> collection, BlueQueryOnDisk<T>
 	@Override
 	public void execute() throws BlueDbException {
 		List<BlueEntity<T>> entities = query.getEntities();
-		List<PendingChange<T>> updates;
+		List<IndividualChange<T>> changes;
 		try {
-			updates = createUpdates(entities, updater);
+			changes = createChanges(entities, updater);
 		} catch(Throwable t) {
 			t.printStackTrace();
 			throw new BlueDbException("Error updating values", t);
 		}
 
+		Collections.sort(changes);
+		PendingBatchChange<T> change = PendingBatchChange.createBatchChange(changes);

```

Comment:
We'll probably need to make these same changes to the replace multiple task once we merge that in